test/e2e/cli_test.go

```diff
@@ -963,9 +963,6 @@ func (s *CLISuite) TestWorkflowLevelSemaphore() {
 			}
 		}).
 		SubmitWorkflow().
-		RunCli([]string{"get", "semaphore-wf-level-1"}, func(t *testing.T, output string, err error) {
```

Comment:
This is another flaky condition.  I will work on enhance e2e to support waitcondition for `runcli`

Reply:
In this test case, we need to submit two workflows and monitor the status of both